src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -8,17 +8,20 @@
 import seedu.address.model.person.NameContainsKeywordsPredicate;
 
 /**
- * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Finds and lists all contacts in address book whose name contains any of the argument keywords.
```

Comment:
There are instances of Persons, instances of Contacts. Just a general comment for our entire code base.